eclipse-scout-core/src/util/BinaryResource.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2010, 2023 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+/**
+ * Warper for binary content with metadata.
```

Comment:
Typo: Wrapper instead of Warper
In General: Use Markdown syntax in JsDoc instead of HTML -> <p> can be removed.